descriptors/ModuleDescriptor-template.json

```diff
@@ -699,6 +699,16 @@
     {
       "id": "configuration",
       "version": "2.0"
+    },
+    {
+      "id": "user-tenants",
+      "version": "1.0"
+    }
+  ],
+  "optional": [
```

Comment:
we do not invoke any consortia endpoints, so this optional interface can be deleted

Reply:
Removed, thanks